### ISSUE TITLE
wrong main property in bower.json

### ISSUE THREAD

**Issue by wrtlbrmpft** [alexandernst/angular-multi-select#53]
The main property in bower.json points to the wrong path.
It should be 'src/angular-multi-select.js'.

**Comment by alexandernst** [comment]
What is wrong with the current one?

**Comment by wrtlbrmpft** [comment]
see https://github.com/bower/spec/blob/master/json.md#main
build tools like wiredep can't find your main file without the correct path to it.

**Comment by alexandernst** [comment]
I still can't understand what exactly is wrong. `dist/angular-multi-select.js` is a completely valid path and the file is exactly the same as in the `src` directory (it's copied from there). Where exactly is the problem? Build tools don't like dirs called `dist` perhaps?

**Comment by wrtlbrmpft** [comment]
i just realized that your bower.json looks correct in the master branch,
but when i install angular-multi-select via
`bower install angular-multi-select-alexandernst`
i get version 5.4.1 and your bower.json looks like this:

``` javascript
{
  "name": "angular-multi-select-alexandernst",
  "version": "5.4.1",
  "homepage": "https://github.com/alexandernst/angular-multi-select",
  "authors": [
    "Alexander Nestorov <alexandernst@gmail.com>"
  ],
  "description": "A multi select dropdown directive for AngularJS",
  "main": "angular-multi-select.js",
  "keywords": [
    "angular",
    "dropdown",
    "multi",
    "select",
    "multiselect"
  ],
  "license": "MIT",
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "test",
    "tests"
  ]
}
```

here the main property points to the wrong path.

**Comment by alexandernst** [comment]
Oh, I see, I'll release a new version to fix that.

**Comment by alexandernst** [comment]
This should be fixed now in 5.5.3
Please close this issue if it's fixed.

**Comment by wrtlbrmpft** [comment]
works.. thank you.
